inception-project/inception
# Feature/1856 support regex based recommendations

**What's in the PR**
* This is a first version just to let you have a look if this is going into the right direction.
* I will add unit tests next

**How to test manually**
* Setup a new Regex Recommender for the Named Entity Layer, and value feature
* Check the "Always active" box.
* Select the document that I have attached. It contains lemma annotations.
* Add a Named Entity annotation
* Enter a regex
* Now you should get recommendations for all the lemmas that fit your regex.

**Automatic testing**

**Documentation**


[Bundestag_08-1.zip](https://github.com/inception-project/inception/files/5966112/Bundestag_08-1.zip)
